core/src/main/scripts/importer/validateData.py

```diff
@@ -3318,12 +3502,15 @@ class GsvaScoreValidator(GsvaWiseFileValidator):
     in R can calculate a GSVA score or GSVA-like score (such as ssGSEA) per sample per gene set.
     """
 
+    def __init__(self, *args, **kwargs):
+        super(GsvaScoreValidator, self).__init__(*args, **kwargs)
```

Comment:
I think this initialiser method can be safely removed; the superclass method is next in the resolution order anyway.

Reply:
All empty constructor methods were removed. Tests run ok.